.buildkite/scripts/common.sh

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
+set -euo pipefail
+
+source .buildkite/scripts/setenv.sh
```

Comment:
**nit**: IIUC, this particular step is already done in the `pre-command` hook. Running twice should not cause any trouble, right?